pom.xml

```diff
@@ -16,7 +16,7 @@
   <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
-    <jenkins.version>2.332.1</jenkins.version>
+    <jenkins.version>2.355-SNAPSHOT</jenkins.version>
```

Comment:
```suggestion
    <!-- TODO https://github.com/jenkinsci/jenkins/pull/6539 -->
    <jenkins.version>2.360-rc32608.d30108d0ed5a_</jenkins.version>
```

This will make the CI build work.